tests/playbooks/tests_states.yml

```diff
@@ -47,3 +47,16 @@
             state: down
     - include_tasks: tasks/assert-device_absent.yml
     - include_tasks: tasks/assert-profile_absent.yml
+
+    # test case (remove non existent profile)
+    # I can set down and remove a non existent profile
+    - name: Set down non_existent profile
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: non_existent
+            state: down
+            persistent_state: absent
+    - include_tasks: tasks/assert-device_absent.yml
```

Comment:
These tasks are actually not working as intended here, since they use the variable `interface` and therefore do not check `non_existent` here.